presto-base-jdbc/src/test/java/io/prestosql/plugin/jdbc/TestH2Jdbc.java

```diff
@@ -13,24 +13,31 @@
  */
 package io.prestosql.plugin.jdbc;
 
-import io.prestosql.testing.AbstractTestQueries;
+import com.google.common.collect.ImmutableList;
 import io.prestosql.testing.QueryRunner;
+import io.prestosql.testing.sql.JdbcSqlExecutor;
 import io.prestosql.tpch.TpchTable;
 
+import java.util.Map;
+import java.util.Properties;
+
 import static io.prestosql.plugin.jdbc.H2QueryRunner.createH2QueryRunner;
 
-public class TestJdbcDistributedQueries
-        extends AbstractTestQueries
+public class TestH2Jdbc
```

Comment:
This is more "test base-jdbc with h2" rather than "test h2 with jdbc", so 

```
TestBaseJdbcWithH2
```
or
```
TestJdbcWithH2
```

similarly, the base class could be: BaseH2JdbcTest -> BaseBaseJdbcTestWithH2 😉  or BaseJdbcTestWithH2